OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs

```diff
@@ -115,8 +115,8 @@ class ImportTSMapCommand : IUtilityCommand
 			{ 0xAF, "trock03" },
 			{ 0xB0, "trock04" },
 			{ 0xB1, "trock05" },
-			{ 0xBB, "veinholedummy" },
-			{ 0xBC, "crate" }
+			//// Skipping the veinhole dummy on purpose.
```

Comment:
Why four `/`'s?

Reply:
To silence StyleCop, which suggested this or add a newline for which the comment is the replacement on purpose.

Reply:
A cleaner fix would be to replace it with `{ 0xB2, null }, // veinhole dummy` and then replace the hardcoded veinhole check in `ReadOverlay` with a null check to skip.

Reply:
Moved the comment into the commit message.